packages/nextra/src/context.ts

```diff
@@ -24,12 +23,12 @@ function filter(
 } {
   let activeLevelPages: Page[] = []
   const items: Page[] = []
-  const meta =
-    pageMap.find((item): item is MetaJsonFile => item.name === META_FILENAME)
-      ?.meta || {}
+  const meta = pageMap.find(
+    (item): item is MetaJsonFile => item.kind === 'Meta'
+  )!.data
```

Comment:
now `_meta.json` fill be always exist in `pageMap` and in `fileMap` even it's not exist on disk, this `_meta.json` will contain all pages with the title as `item.frontMatter.title` or `item.name`